.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,28 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
```

Comment:
Consider specifying an explicit version here -- I can't find the exact rules for what `3.x` means, but if it means "the most recent version", it might end up using a version we don't explicitly support yet.  For instance they already added python 3.9 to GH actions but we aren't formally testing on it yet.